.github/renovate.json5

```diff
@@ -5,6 +5,8 @@
     // https://github.com/newrelic/nri-kubernetes/issues/205
     ":disableDependencyDashboard"
   ],
+  // Label PRs with `dependencies`.
+  "labels": ["dependencies"],
```

Comment:
1. Why not using the label `Skip Changelog` and avoid creating the label `dependencies`? The simpler the label system, the easier will be to maintain.

2. Both Dependabot and Renovate create dependency PRs. Is it easy to make this change for Dependabot as well?

3. We should also document this on the `CONTRIBUTING.md` file to let developers know of these labels, like `ci/skip-e2e` and `ci/skip-load-test`. Personally, I have no idea what `ci/skip-load-test` does.

Reply:
1. I believe it's pretty standard to apply "dependencies" as a label for any prs that are dependabot/renovatebot dependency bump prs so I'm hesitate to remove that label.
2. Dependabot already creates a "dependencies" label.
3. The e2e tests are skipped when a label `ci/skip-e2e` is added. Not sure where `skip-load-test` is used. But I can update both the contributing.md file and changelog workflow to mention adding "Skip Changelog" label.

Reply:
Sounds good to me